spec/features/workflow/publish_without_review_spec.rb

```diff
@@ -1,11 +1,14 @@
 # frozen_string_literal: true
 
-RSpec.feature "Publish without review" do
+RSpec.feature "Publish without review", inline: true do
```

Comment:
It wasn't until I reached the spec file that I realised inline true was for sidekiq. Perhaps call it sidekiq_inline? Could also be that we want it to default to sidekiq inline for all feature tests as I can't imagine we'd want it it not be inline for feature tests.

Reply:
I'm used to having Sidekiq inline in all tests, but we've actually got a use case for that: the `schedule_spec`. I'd like to keep Sidekiq out of the tests if possible, so how about one of: `background` or `async`?

Reply:
Oh yes - I remember that one. Darn, I'd forgotten that test. We can set that one inline but from my googling it'll behave a bit weird and run straight away: https://github.com/mperham/sidekiq/issues/2255

Maybe it's best to set all feature tests to run inline and for that (and any other ones) to have a before block of `before { Sidekiq::Testing.fake! }`.

I don't think any option that is provided just for sidekiq can be anything but opaque unless it mentions sidekiq in it.

Reply:
Yeah, I think I tried that already, and I the fact it behaves weird with scheduling wasn't surprising, really. I've changed to run everything inline except that one spec.